### PR TITLE
test zap scan

### PR DESCRIPTION
**Pull request by st3v3nhunt**
https://eaflood.atlassian.net/browse/PSD-946

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,12 +36,12 @@ for
...
 [azure-identity-binding](./helm/ffc-demo-claim-service/templates/azure-identity-binding.yaml).
 
-| Name                               | Description                                                                                  |
-| ---------------------------------- | -------------------------------------------------------------------------------------------- |
-| MESSAGE_QUEUE_HOST                 | Azure Service Bus hostname, e.g. `myservicebus.servicebus.windows.net`                       |
-| MESSAGE_QUEUE_PASSWORD             | Azure Service Bus SAS policy key                                                             |
-| MESSAGE_QUEUE_SUFFIX               | Developer specific queue suffix to prevent collisions, only required for local development   |
-| MESSAGE_QUEUE_USER                 | Azure Service Bus SAS policy name, e.g. `RootManageSharedAccessKey`                          |
+| Name                   | Description                                                                                |
+| ----                   | -----------                                                                                |
```
Comment (johnwatson484) [inline README.md:40]: Does this whitespace need to be here?

**Review by johnwatson484 — COMMENTED** [review]
Looks good, just a question about the addition of whitespace

File: README.md (hunk 1 above)
Comment (st3v3nhunt) [reply README.md:40]: I use a plugin to format tables and keeping the columns equal width is one of the things it does, improving readability. Removing the `-` from the heading makes the column fit to the content when it is formatted as opposed to having to be at least as wide the set of `-` that were initially created.

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**